File: iOS/Code/Xamarin/bugTrap/bugTrapKit/Controllers/TableViewControllers/CollectionViewCells/BtEmbeddedImageCollectionViewCell.cs
using System;

using UIKit;

namespace bugTrapKit
{
	public partial class BtEmbeddedImageCollectionViewCell : UICollectionViewCell
	{
		public BtEmbeddedImageCollectionViewCell (IntPtr handle) : base(handle)
		{
		}

		public void SetData (UIImage image)
		{
			ImageView.Image = image;
		}
	}
}

File: iOS/Code/Swift/bugTrap/bugTrapKit/Controllers/TableViewControllers/CollectionViewCells/BtEmbeddedImageCollectionViewCell.swift
//
//  BtEmbeddedImageCollectionViewCell.swift
//  bugTrap
//
//  Created by Colby L Williams on 6/30/14.
//  Copyright (c) 2014 bugTrap. All rights reserved.
//

import UIKit

class BtEmbeddedImageCollectionViewCell: UICollectionViewCell {
    
	@IBOutlet weak var ImageView: UIImageView!
	
	override func awakeFromNib() {
		super.awakeFromNib()
		
//		layer.borderWidth = 0.5
//		layer.borderColor = Colors.LightGray.cgColor
	}
	
	func setData(image: UIImage?)
	{
		ImageView.image = image
	}
}